src/Read_Forcing.f90

```diff
@@ -13,77 +13,75 @@ SUBROUTINE Read_Forcing
 no_flow=0
 no_heat=0
 do nr=1,nreach
-  do nc=1,no_cells(nr)-1 
-    no_flow=no_flow+1
-    no_heat=no_heat+1
-!
-    nrec_flow=flow_cells*(ndays-1)+no_flow
-    nrec_heat=heat_cells*(ndays-1)+no_heat
-!
-    read(35,'(2i5,2f10.1,2f6.1,f7.1,f6.2)' &
-           ,rec=nrec_flow) nnd,ncell &
-           ,Q_in(no_heat),Q_out(no_heat),Q_diff(no_heat) &  
-           ,depth(no_heat),width(no_heat),u(no_heat)
-
-!
-    if(u(no_heat).lt.0.01) u(no_heat)=0.01
-    if(ncell.ne.no_heat) write(*,*) 'Flow file error',ncell,no_heat 
-!
-    read(36,'(i5,2f6.1,2f7.4,f6.3,f7.1,f5.1)' &
-           ,rec=nrec_heat) ncell &
-           ,dbt(no_heat),ea(no_heat) &
-           ,Q_ns(no_heat),Q_na(no_heat),rho &
+  do nc=1,no_cells(nr)-1 
+    no_flow=no_flow+1
+    no_heat=no_heat+1
+!
+    nrec_flow=flow_cells*(ndays-1)+no_flow
+    nrec_heat=heat_cells*(ndays-1)+no_heat
+!
+
+    read(35,*) nnd,ncell &
```

Comment:
I changed this line when reading flow and energy files, because I split lines in those input files